blogs/lokiless_netobserv/index.md

```diff
@@ -0,0 +1,91 @@
+# Network Observability without Loki
+
+By: Mehul Modi, Steven Lee
+
+Recently, Network Observability operator 1.6 released a major enhancement to provide network insights for your OpenShift cluster without Loki. This enhancement was also featured in [What's new in Network Observability 1.6](../whats_new_1.6) blog providing quick overview of the feature. In this blog, lets look at some of the advantages and trade-offs users would have when deploying network observability with Loki disabled.
+
+# Configure Network Observability without Loki
+Loki as datasource is currently enabled by default. To configure Network Observability operator without Loki, when configuring Flowcollector resource simply set `.spec.loki.enable` to `false`
+
+```yaml
+loki:
+  enable: false
+```
```

Comment:
You could also add some context, such as telling that, without Loki, metrics are still being sent to Prometheus - and since Prometheus is in core OpenShift payload, you don't need further backend configuration.